pyproject.toml

```diff
@@ -304,7 +304,6 @@ exclude = [
     "^llama_stack/strong_typing/inspection\\.py$",
     "^llama_stack/strong_typing/schema\\.py$",
     "^llama_stack/strong_typing/serializer\\.py$",
```

Comment:
probably need to add /starter/starter.py here as well?

Reply:
@raghotham no we don't -- that's a mypy exclusion list. we fixed the mypy typing error :)